src/styles/theme/fonts.ts

```diff
@@ -0,0 +1,63 @@
+export const oakAllFontSizes = {
+  "font-size-1": 12,
+  "font-size-2": 14,
+  "font-size-3": 16,
+  "font-size-4": 18,
+  "font-size-5": 20,
+  "font-size-6": 24,
+  "font-size-7": 32,
+  "font-size-8": 40,
+  "font-size-9": 48,
+  "font-size-10": 56,
+};
+
+export type FontWeight = 300 | 400 | 600 | 700;
```

Comment:
they are used in the parseFont.tsx functions

Reply:
Only because the return type is unnecessarily specified from each function. Remove those and there's no need to import the types. Could you make it so ?